app/controllers/users_controller.rb

```diff
@@ -6,7 +6,7 @@ def auth_user
   end
 
   def show
-    @user = current_user
+    @user = User.find(params[:id])
```

Comment:
@KateCatlin Are you okay if anyone logged in can see *anyone's* profile as long as they have the id for the url? Or should a user only be allowed to see someone else's profile if they are in their current peer group?